app/admin/withdrawal.rb

```diff
@@ -0,0 +1,35 @@
+ActiveAdmin.register Withdrawal do
+  permit_params :shop_id, :amount, :approved
+
+  index do
+    selectable_column
+    id_column
+    column :shop_id
+    column :amount
+    column :approved
+    column :created_at
+    column :updated_at
+    actions
+  end
+
+  filter :id
+  filter :shop_id
+  filter :amount
+  filter :approved
+  filter :created_at
+  filter :updated_at
+
+  show :title => :id do
+    attributes_table do
+      row :id
+      row :shop_id
+      row :amount
+      row :approved
```

Comment:
OK. Another issue: I don't know why the amount in database is stored with the unit of cent – € 5 will be 500 in it. I don't know how to fix this in terms of showing the correct value.